custom_components/tuya_local/devices/jiahong_et72w_thermostat_f.yaml

```diff
@@ -0,0 +1,225 @@
+name: Jiahong ET-72W Thermostat (F)
+primary_entity:
+  entity: climate
+  dps:
+    - id: 101
+      type: boolean
+      name: power
+      mapping:
+        - dps_val: false
+          value: "off"
+      hidden: true
+    - id: 102
+      type: integer
+      name: temperature
+      range:
+        min: 120
```

Comment:
Does this range come from a manual or tuya cloud api, or did you calculate it yourself?
It doesn't seem right, as it equates to a range of -12C - 24C with a step of 2C.  If you calculated yourself, I can understand that you might have forgotten to add the 32 degree offset at the end, and also may have inverted somewhere in the step calculation (normally devices with 0.5C step have 1F steps).  If it came from docs, then maybe the device designers made these mistakes, making the device practically unusable in F, but in that case the config should reflect the actual device limits.
My guess would be a 400-1000 range with 10 step, which gives 4-38, not quite exactly matching the C range but with round numbers that are very close.

Reply:
Hi @make-all

The range on the C configuration is correct. 
I messed up the conversion to F. 
I'll raise another PR with the corrected range and include the test cases when I have a chance.
Thanks for catching this

Reply:
Hi @make-all 

I just reviewed your PR's to merge the configs into a single one. I had tried that but couldn't get the secondary entities to work correctly. I see how you did it an know what I did wrong.